docs/make.jl

```diff
@@ -140,8 +140,8 @@ makedocs(sitename="TrixiParticles.jl",
                      "Util" => joinpath("general", "util.md")
                  ],
                  "Systems" => [
-                     "Discrete Element Method (Solid)" => joinpath("systems",
-                                                                   "dem.md"),
+                     "Fluid Models" => joinpath("systems", "fluid.md"),
+                     "Discrete Element Method (Solid)" => joinpath("systems", "dem.md"),
```

Comment:
Would it make more sense to make it like this:
- Systems
  - Fluid Models
    - Overview (with the new content that you added in "fluid.md")
    - WCSPH
    - EDAC
  - DEM

Now we have "Fluid Models" on the same level as "WCSPH" and "EDAC", which is confusing to me, as these are fluid models.